### PR TITLE
fix(pipelines)change label selector

### PR DESCRIPTION
**Pull request by abhinandan13jan**
#Bugfix for https://jira.coreos.com/browse/ODC-661
Carries changes in label selector for filtering runs for a pipeline.
@christianvogt @rohitkrai03 Please review

To test this 
1. Run frontend/extend/devconsole/components/mocks/pipelines/install_pipeline_mocks,sh
Alternately, from frontend/extend/devconsole/components/mocks/pipelines
run the following
oc delete pipelines --all
oc delete pipelineruns --all
oc delete tasks --all
oc delete taskruns --all
oc create -f  base.yaml
oc create -f resources.yaml

2. Click on pipeline resource link on sidenav, you'll get a view as follows
Screenshot from 2019-05-02 14-14-24

3. Click on the Pipeline and visit the 'Pipeline Runs' tab, you'll get a view as follows
Screenshot from 2019-05-02 14-14-40

### CODE REVIEW

**Comment by rohitkrai03** [comment]
@abhinandan13jan What is this change actually fixing? Can you give a bit more context on why this change needs to go in?

**Comment by abhinandan13jan** [comment]
@rohitkrai03 I ve changed the label selector from "pipeline" to "tekton.dev/pipeline" as suggested by the tekton team.

**Comment by sspeiche** [comment]
Fixes bug https://jira.coreos.com/browse/ODC-661, right?

**Comment by abhinandan13jan** [comment]
@sspeiche  Yes, @rohitkrai03  Currently Pipelineruns are filtered using label "pipeline" which has to be added manually, instead we need to use label "tekton.dev/pipeline" which comes in as a default hook.

**Comment by christianvogt** [comment]
Maybe I'm missing something but my pipeline runs is still empty. Can anyone else verify this?
cc @rohitkrai03

**Comment by abhinandan13jan** [comment]
@christianvogt Please re run with new mock data. I've updated the script as well as the data. Run the script again. The webhook is not auto applying the label on the pipelinerun all the time. Running the script twice in the fail cases worked for me.(Web hook is not an issue over UI end for sure)

File: frontend/public/extend/devconsole/components/pipelines/PipelineRuns.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ interface PipelineRunsProps {
 class PipelineRuns extends React.Component<PipelineRunsProps> {
   render() {
     const selector = {
-      pipeline: this.props.obj.metadata.name,
+      "tekton.dev/pipeline": this.props.obj.metadata.name,
```
Comment (christianvogt) [inline frontend/public/extend/devconsole/components/pipelines/PipelineRuns.tsx]: Lint error. `Strings must use singlequote`.

**Comment by sbose78** [comment]
test

**Comment by abhinandan13jan** [comment]
@christianvogt Fixed lint error and added.

**Review by rohitkrai03 — APPROVED** [review]
LGTM

**Pull request merged**